#latitude is in degree
float64 latitude
#longitude is in degree
float64 longitude
#relative height in meters
float64 target_height
#heading to next waypoint in [-180,180]
int16 heading
#stay time is in second
uint16 staytime
